media/js/firefox/sticky-promo.js

```diff
@@ -30,6 +37,10 @@
         // Open promo
         Mzp.StickyPromo.open();
 
+        // Add modifier class to the footer to make sure the language selection drop-down is not obscured by the sticky promo
+        var footer = document.querySelector('.mzp-c-footer');
+        footer.classList.add('sticky-overlay');
```

Comment:
Might be worth checking that a page has a footer before attempting to modify the class?

```javascript
if (footer) {
    footer.classList.add('intersect-sticky-overlay');
}

Reply:
Added! 

Side question - because we know which pages this promo is being added to, isn't it assumed a footer is there?

Reply:
The footer is there for all pages currently, but this seems like a small enough change to cater for unanticipated futures.